tests/phpt/error_handler_captures_errors_not_silencable_on_php_8_and_up.phpt

```diff
@@ -55,9 +55,11 @@ $transportFactory = new class implements TransportFactoryInterface {
 
 error_reporting(E_ALL & ~E_USER_ERROR);
 
-$client = ClientBuilder::create(['error_types' => E_ALL, 'capture_silenced_errors' => false])
-    ->setTransportFactory($transportFactory)
-    ->getClient();
+$client = ClientBuilder::create([
+    'dsn' => 'http://public@example.com/sentry/1',
+    'error_types' => E_ALL,
+    'capture_silenced_errors' => false
+])->setTransportFactory($transportFactory)->getClient();
```

Comment:
Nitpicking: this looks a bit messy to read, what if we extracted the options into its own `$options` variable so that we can keep the formatting as before with each method call on its own line?

Reply:
Yes, will do that.